hack/run-wsu-ci-e2e-test.sh

```diff
@@ -20,8 +20,11 @@ fi
 # The WSU playbook requires the cluster address, we parse that here using oc
 CLUSTER_ADDR=$(oc cluster-info | head -n1 | sed 's/.*\/\/api.//g'| sed 's/:.*//g')
 
+# Set up hybrid networking on the cluster, a requirement of OVNKubernetes on Windows
+oc get Network.operator.openshift.io cluster -o json | jq '.spec.defaultNetwork.ovnKubernetesConfig = {"hybridOverlayConfig":{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}' | oc apply -f -
+
 # Run the test suite
 cd $TEST_DIR
-GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on CLUSTER_ADDR=$CLUSTER_ADDR WSU_PATH=$WMCO_ROOT/tools/ansible/tasks/wsu/main.yaml go test -v -timeout 20m .
+GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on CLUSTER_ADDR=$CLUSTER_ADDR WSU_PATH=$WMCO_ROOT/tools/ansible/tasks/wsu/main.yaml go test -v -timeout 30m .
```

Comment:
I see this being increased in every PR. Please say why you are doing this in your commit message.